### PR TITLE
Update deps - XCM related types

### PR DESCRIPTION
**Pull request by joelamouche**

### CODE REVIEW

File: packages/apps-config/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "@zeitgeistpm/type-defs": "0.2.7",
     "@zeroio/type-definitions": "0.0.6",
     "lodash": "^4.17.21",
-    "moonbeam-types-bundle": "1.2.5",
+    "moonbeam-types-bundle": "latest",
```
Comment (jacogr) [inline packages/apps-config/package.json:32]: Would prefer a specific version here. This package is used by others as well.

Comment (joelamouche) [reply packages/apps-config/package.json]: yes ofc sorry about that

**Review by jacogr — DISMISSED** [review]

**Comment by jacogr** [comment]
I think the “bump all” here is. Resting issues with the build. Just bump the moonbeam package.

**Comment by jacogr** [comment]
Closing, used `yarn upgrade-interactive` to bump all bundle deps (a weekly task), in a different (now merged) PR.

Sadly, just allowing yarn to bump-at-will yields issue after issue (like in this PR), so I always curate and pin-point.

**Pull request closed without merging**